applications/ConstitutiveLawsApplication/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ file(GLOB_RECURSE KRATOS_CONSTITUTIVE_LAWS_APPLICATION_CORE
     ${CMAKE_CURRENT_SOURCE_DIR}/constitutive_laws_application.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/constitutive_laws_application_variables.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive/*.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_constitutive_thermal/*.cpp
```

Comment:
For the sake of consistency with the rest of applications, I'd suggest to put the new constitutive models together with their corresponding base classes within the `custom_constitutive` folder.